testing/functional/tests/cypress/integration/student/01-welcome-page.spec.ts

```diff
@@ -0,0 +1,47 @@
+import WelcomeObject from "../../page-objects/student-objects/WelcomeObject"
```

Comment:
It seems that the files are still missing formatting. For instance, the formatted is supposed to add a ";" to the end of all the lines automatically and remove the extra spaces. Please check all the files.

Reply:
In the latest you will find the code in formatted and at the end of lines ";" has been added.